docs/index/secondary-index-guide.md

```diff
@@ -188,4 +188,13 @@ where we have old stores.
 Syntax
   ```
   REGISTER INDEX TABLE index_name ON [TABLE] [db_name.]table_name
+  ```
+
+### Repair index Command
+This command is used to reload segments in the SI table in case when there is some mismatch in the number
+of segments in main table and the SI table
+
+Syntax
+  ```
+  REPAIR INDEX ON TABLE maintable_name
```

Comment:
Looks like index_name is missing from syntax.

Reply:
done